systemtest/tmt/plans/main.fmf

```diff
@@ -118,7 +118,7 @@ prepare:
       # Setup local registry for building images and connect image
       export DOCKER_REGISTRY="${HOST}:5001"
       export DOCKER_ORG="strimzi"
-      export DOCKER_TAG="test"
+      export DOCKER_TAG="latest"
```

Comment:
why is this needed?

Reply:
Because with `test` KafkaBridge tests were failing. With the
```
sed -i "s#quay.io/strimzi/kafka#${DOCKER_REGISTRY}/${DOCKER_ORG}/kafka#g" packaging/install/cluster-operator/*-Deployment-strimzi-cluster-operator.yaml
```
we were also modifying the content of the `KafkaBridge` image instead of just replacing Kafka images. So in other words when I fixed the image issue then 
```java
quay.io/strimzi/kafka-bridge:test
```
was another problem. So I have changed the `test` docker tag to the `latest` and we would use:
1. Kafka/operator etc. images with replaced DOCKER_REGISTRY (e.g., `10.2.4.1:5001/strimzi/kafka:latest-kafka-3.9.0`)
2. but KafkaBridge will be used the latest  (e.g.,`quay.io/strimzi/kafka-bridge:latest`)

Reply:
wouldn't be enought just specify env var for bridge image?

Reply:
Yeah that's why I wanted to have it changed as well. Because everywhere we have `latest` and just here it's `test`.

Reply:
The tag was to ensure that pipeline won't download pre-built images from quay. If you think this cannot hapen, I am fine with changing it.

Reply:
I'm not familiar with the pipeline, but I guess we can do similar thing as on AZP? We are building images into private registries. I think we can do similar thing here, so it will not match the official ones?

Reply:
> We are building images into private registries. I think we can do similar thing here, so it will not match the official ones?

Yeah, we are doing it.
https://github.com/strimzi/strimzi-kafka-operator/blob/74f882b750c7e1afbd97062745cbfe494446672f/systemtest/tmt/plans/main.fmf#L101-L146

> The tag was to ensure that pipeline won't download pre-built images from quay

I think this ensure it or?
https://github.com/strimzi/strimzi-kafka-operator/blob/74f882b750c7e1afbd97062745cbfe494446672f/systemtest/tmt/plans/main.fmf#L119

Reply:
ye, this ensures it as well

Reply:
Then you don't have to do anything :) and it should work, yes